Automattic/wp-calypso
# Update the name of the breakpoint mixin to show that its deprecated

#### Changes proposed in this Pull Request

* Based on p4TIVU-9pb-p2 let's rename this mixin to make it clear that it's deprecated
* I looked into adding a stylelint rule, but it looked complicated so I took the easy route and just renamed the mixin :)


The reason behind this:
- We have virtually the same breakpoints in Calypso and Gutenberg but using different code.
- By using the same breakpoints we'll keep our CSS smaller and have a more consistent experience.
- We also keep our code simpler.

#### Testing instructions

* Look for uses of this mixin that I missed (I did a find and replace on `include breakpoint`);
* Test some pages to make sure they are still responsive.
* Check that the docs change is helpful
